widgets/icon/styles/sow-icon.less

```diff
@@ -0,0 +1,14 @@
+@color: h1;
```

Comment:
Ideally, these defaults should make sense. So the actual default color as `@color`. Same for `@size`. Can't remember if this has any benefit, but it just feels neater.